.github/CHANGELOG.md

```diff
@@ -36,10 +36,16 @@
 * Added `sympy>=1.5` to the list of dependencies.
   Removed the `sympy.functions.atan2` workaround now that SymPy has been fixed.
   [#280](https://github.com/XanaduAI/strawberryfields/pull/280)
-  
+
 * Removed two unnecessary else statements that pylint complained about.
   [#290](https://github.com/XanaduAI/strawberryfields/pull/290)
-  
+
+* Fixed a bug in the `MZgate`, where the internal and external phases were
+  in the wrong order in both the docstring and the argument list. The new
+  signature is `MZgate(phase_in, phase_ex)`, matching the existing hardware
```

Comment:
What do you mean `hardware decomposition`? There were two definitions directly in SF, no?

Reply:
Was referring to the `rectangular_symmetric` decomposition, will change it!
```suggestion
  signature is `MZgate(phase_in, phase_ex)`, matching the existing `rectangular_symmetric`
```